css-layout-classes/styles.css

```diff
@@ -0,0 +1,36 @@
+.container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  margin-bottom: 8px;
+}
+
+.column-one-third {
+  width: calc(100% / 3);
+}
+
+img {
+  width: 100%;
+  height: 100%;
```

Comment:
I would remove the height here, you just want to set either width or height for images, and in this case, width is what you want. That will allow the image to automatically maintain the correct aspect ratio.